services/src/handlers/layers.rs

```diff
@@ -535,6 +542,100 @@ async fn layer_to_workflow_id_handler<C: Context>(
     Ok(web::Json(workflow_id))
 }
 
+/// Persist a raster layer from a provider as a dataset.
+#[utoipa::path(
+    tag = "Layers",
+    post,
+    path = "/layers/{provider}/{layer}/dataset",
+    responses(
+        (status = 200, description = "Id of created task", body = TaskResponse,
+            example = json!({
+                "task_id": "7f8a4cfe-76ab-4972-b347-b197e5ef0f3c"
+            })
+        )
+    ),
+    params(
+        ("provider" = DataProviderId, description = "Data provider id"),
+        ("layer" = LayerCollectionId, description = "Layer id"),
+    ),
+    security(
+        ("session_token" = [])
+    )
+)]
+async fn layer_to_dataset<C: Context>(
+    session: AdminSession,
+    ctx: web::Data<C>,
+    path: web::Path<(DataProviderId, LayerId)>,
+) -> Result<impl Responder> {
+    let (provider, item) = path.into_inner();
+
+    let layer = match provider {
+        crate::datasets::storage::DATASET_DB_LAYER_PROVIDER_ID => {
+            ctx.dataset_db_ref().get_layer(&item).await?
+        }
+        crate::layers::storage::INTERNAL_PROVIDER_ID => ctx.layer_db_ref().get_layer(&item).await?,
+        _ => {
+            ctx.layer_provider_db_ref()
+                .layer_provider(provider)
+                .await?
+                .get_layer(&item)
+                .await?
+        }
+    };
+
+    let session = C::Session::from(session.clone());
+
+    let execution_context = ctx.execution_context(session.clone())?;
+
+    let raster_operator = layer
+        .workflow
+        .clone()
+        .operator
+        .get_raster()?
+        .initialize(&execution_context)
+        .await?;
+
+    let result_descriptor = raster_operator.result_descriptor();
+
+    let qr = QueryRectangle {
+        spatial_bounds: result_descriptor
+            .bbox
+            .ok_or(Error::InvalidLayerResultDescriptor {
```

Comment:
we should discuss this. For me this is reasonable but the ResultDescriptor is not invalid since that fields are options.

Reply:
I guess we can rename it to LayerResultDescriptorMissingFields or instead of an error have some sensible default values. I would need guidance for the latter.

Reply:
lets call it "LayerResultDescriptorMissingFields" and then we need a new task / card where we change that a raster result descriptor always has a bound. Currently we know bounds in all cases i know off. If we really don't know the bounds we can use the bounds of the SRS as fallback.

Reply:
974d065ddd83e975f5edb25342c127f44007ba1a